app/src/main/java/org/fossasia/phimpme/utilities/RetrofitClient.java

```diff
@@ -0,0 +1,28 @@
+package org.fossasia.phimpme.utilities;
+
+import okhttp3.OkHttpClient;
+import okhttp3.logging.HttpLoggingInterceptor;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+/** Created by @codedsun on 19/Oct/2019 */
+
+// Retrofit client used for API Calls
+public class RetrofitClient {
+
+  private static OkHttpClient.Builder httpClient() {
+    OkHttpClient.Builder httpClient = new OkHttpClient.Builder();
+    HttpLoggingInterceptor interceptor = new HttpLoggingInterceptor();
+    interceptor.level(HttpLoggingInterceptor.Level.BODY);
```

Comment:
Level BODY is too verbose

Reply:
@iamareebjamal Just used for development purpose, will add check of debug and production here from buildconfig

Reply:
BODY is too verbose even for debug. Stetho should be used instead